website/src/components/SearchPage/DownloadDialog/DownloadDataType.ts

```diff
@@ -0,0 +1,39 @@
+export type DownloadDataType =
+    | { type: 'metadata' }
+    | { type: 'unalignedNucleotideSequences'; segment?: string }
+    | { type: 'alignedNucleotideSequences'; segment?: string }
+    | { type: 'alignedAminoAcidSequences'; gene: string };
+
+/**
+ * Get a shortened kebab-case datatype including the gene and sequence as well.
+ */
+export const dataTypeForFilename = (dataType: DownloadDataType): string => {
+    switch (dataType.type) {
+        case 'metadata':
+            return 'metadata';
+        case 'unalignedNucleotideSequences':
+            return dataType.segment !== undefined ? `nuc-${dataType.segment}` : 'nuc';
```

Comment:
```suggestion
            // segment is undefined in case of single segmented (not e.g. main)
            return dataType.segment !== undefined ? `nuc-${dataType.segment}` : 'nuc';
```